lib/miq_automation_engine/service_models/miq_ae_service_service_template_container_template.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceServiceTemplateContainerTemplate < MiqAeServiceServiceTemplateGeneric
+    expose :container_template
```

Comment:
do we see the need to use `container_template` in the automate? If not immediately, let's hold off this PR because the ae model can be automately generated now.